eng/pipelines/bicep-warnings.yml

```diff
@@ -0,0 +1,46 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - '**/*.bicep'
+      - eng/pipelines/bicep-warnings.yml
+
+stages:
+  - stage: LintBicepFiles
+    displayName: Lint Bicep Files
+
+    pool:
+      vmImage: azsdk-pool-mms-ubuntu-2204-1espt
+
+    jobs:
+      - job: LintBicep
+        timeoutInMinutes: 20
+        steps:
+          - script: |
+              az bicep install
+              find . -name '*.bicep' -exec az bicep lint --file {} \; > lint-warnings.txt 2>&1
```

Comment:
Does this mean that all output get redirected to the result file, and we will not able to see result in the build web UI? Maybe add a `cat lint-warnings.txt`?

Reply:
Good point, added in https://github.com/Azure/azure-sdk-for-js/pull/30904/commits/c62f166432ab5a59465eb85726c5661502e854c5